libs/experimental/langchain_experimental/data_anonymizer/presidio.py

```diff
@@ -103,18 +124,28 @@ def add_operators(self, operators: Dict[str, OperatorConfig]) -> None:
 
 
 class PresidioAnonymizer(PresidioAnonymizerBase):
-    def _anonymize(self, text: str) -> str:
+    def _anonymize(self, text: str, language: Optional[str]) -> str:
```

Comment:
`None` value does not tell that the english language would be used - imo better default value would be just `'en'`

Reply:
What if only Spanish is provided? That's why I used:
```
if language is None:
    language = self.supported_languages[0]
```

Reply:
then don't make a default value :p `Explicit is better than implicit`